Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -246,25 +242,12 @@ extension IntroViewController {
         // presented version happily rotates with the iPad orientation.
         return .portrait
     }
-}
-
-// MARK: - NotificationThemeable and Notifiable
-extension IntroViewController: NotificationThemeable, Notifiable {
-    func handleNotifications(_ notification: Notification) {
-        switch notification.name {
-        case .DisplayThemeChanged:
-            applyTheme()
-        default:
-            break
-        }
-    }
 
+    // MARK: - Themable
     func applyTheme() {
-        let theme = BuiltinThemeName(rawValue: LegacyThemeManager.instance.current.name) ?? .normal
-
-        let indicatorColor = theme == .dark ? UIColor.theme.homePanel.activityStreamHeaderButton : UIColor.Photon.Blue50
-        pageControl.currentPageIndicatorTintColor = indicatorColor
-        view.backgroundColor = LegacyThemeManager.instance.current.onboarding.backgroundColor
+        pageControl.currentPageIndicatorTintColor = themeManager.currentTheme.colors.actionPrimary
+        pageControl.pageIndicatorTintColor = themeManager.currentTheme.colors.actionSecondary
+        view.backgroundColor = themeManager.currentTheme.colors.layer2
```

Comment:
nit: would be easier to read/short if we set `let theme = themeManager.currentTheme` at the top and use that to set colors instead of calling `themeManager.currentTheme` each time.